src/main/java/it/pagopa/ecommerce/commons/domain/pojos/BaseTransactionRefunded.java

```diff
@@ -0,0 +1,52 @@
+package it.pagopa.ecommerce.commons.domain.pojos;
+
+import it.pagopa.ecommerce.commons.documents.TransactionRefundedData;
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+/**
+ * <p>
+ * POJO for a refund transaction.
```

Comment:
```suggestion
 * POJO for a refunded transaction.
```

Reply:
Fixed in [b43bed1](https://github.com/pagopa/pagopa-ecommerce-commons/pull/31/commits/b43bed13b0d072f42e603cca2e0b4a0ea3012949)